### PR TITLE
Update acl.xml

### PR DESCRIPTION
**Pull request by efexc**
It will be both instructive for people who are just learning how to create a server and wont create security risks if its forgotten to be deleted/changed.

### CODE REVIEW

**Comment by TracerDS** [comment]
> It will be both instructive for people who are just learning how to create a server and wont create security risks if its forgotten to be deleted/changed.

This **will** create permission issues

File: Server/mods/deathmatch/acl.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
       <acl name="SuperModerator" />
       <acl name="Admin" />
       <acl name="RPC" />
-      <object name="user.Console" />
```
Comment (CrosRoad95) [inline Server/mods/deathmatch/acl.xml:32]: Why? "Console" is build in account that you can not login, it is used just for Console

**Comment by efexc** [comment]
> > It will be both instructive for people who are just learning how to create a server and wont create security risks if its forgotten to be deleted/changed.
> 
> This **will** create permission issues

How?

**Comment by TracerDS** [comment]
> > > It will be both instructive for people who are just learning how to create a server and wont create security risks if its forgotten to be deleted/changed.
> > 
> > 
> > This **will** create permission issues
> 
> How?

**Console** is a server side console. It should **NOT** be removed

**Comment by TracerDS** [comment]
Also commenting it is pointless here. If you dont know what you (I mean, you as an owner of the server) are doing in the acl.xml file then maybe you should not edit it and learn about it instead

**Pull request closed without merging**